skrub/tests/_test_dispatch.py

```diff
@@ -0,0 +1,40 @@
+import pytest
```

Comment:
is there a reason to start the name of the file by `_`?

Reply:
Is the file going to be discovered automatically by `pytest`?

Reply:
no reason, that's a typo :)